app/server.py

```diff
@@ -48,7 +48,7 @@ def search(search_engine):
         engine = search_engine
         if engine not in ('google', 'bing', 'duckduckgo', 'yahoo', 'ask',
                           'yandex', 'ubaidu', 'exalead', 'quora', 'tyoutube',
-                          'parsijoo', 'mojeek', 'vdailymotion'):
+                          'parsijoo', 'mojeek', 'vdailymotion', 'hqihooso'):
```

Comment:
#327 must be fixed before we propagate more workarounds.

Reply:
No need of this

Reply:
Please change "hqihooso" to "qihooso"